openconcept/mission/dymos_phases.py

```diff
@@ -257,23 +245,21 @@ def setup(self):
         self.declare_partials(['fltcond|CL'], ['ac|geom|wing|S_ref'], rows=arange, cols=np.zeros(nn))
 
     def compute(self, inputs, outputs):
-        g = 9.80665 #m/s^2
-        outputs['fltcond|CL'] = inputs['fltcond|cosgamma']*g*inputs['weight']/inputs['fltcond|q']/inputs['ac|geom|wing|S_ref']
+        outputs['fltcond|CL'] = inputs['fltcond|cosgamma']*GRAV_CONST*inputs['weight']/inputs['fltcond|q']/inputs['ac|geom|wing|S_ref']
 
     def compute_partials(self, inputs, J):
-        g = 9.80665 #m/s^2
-        J['fltcond|CL','weight'] = inputs['fltcond|cosgamma']*g/inputs['fltcond|q']/inputs['ac|geom|wing|S_ref']
-        J['fltcond|CL','fltcond|q'] = - inputs['fltcond|cosgamma']*g*inputs['weight'] / inputs['fltcond|q']**2 / inputs['ac|geom|wing|S_ref']
-        J['fltcond|CL','ac|geom|wing|S_ref'] = - inputs['fltcond|cosgamma']*g*inputs['weight'] / inputs['fltcond|q'] / inputs['ac|geom|wing|S_ref']**2
-        J['fltcond|CL','fltcond|cosgamma'] = g*inputs['weight']/inputs['fltcond|q']/inputs['ac|geom|wing|S_ref']
+        J['fltcond|CL','weight'] = inputs['fltcond|cosgamma']*GRAV_CONST/inputs['fltcond|q']/inputs['ac|geom|wing|S_ref']
+        J['fltcond|CL','fltcond|q'] = - inputs['fltcond|cosgamma']*GRAV_CONST*inputs['weight'] / inputs['fltcond|q']**2 / inputs['ac|geom|wing|S_ref']
+        J['fltcond|CL','ac|geom|wing|S_ref'] = - inputs['fltcond|cosgamma']*GRAV_CONST*inputs['weight'] / inputs['fltcond|q'] / inputs['ac|geom|wing|S_ref']**2
+        J['fltcond|CL','fltcond|cosgamma'] = GRAV_CONST*inputs['weight']/inputs['fltcond|q']/inputs['ac|geom|wing|S_ref']
 
 class DymosSteadyFlightODE(om.Group):
```

Comment:
Other than `DymosSteadyFlightODE`, the classes in this file are duplicates of `phases.py`. Also, I believe this dymos component is not used anywhere, so we should either remove this file, or only keep `DymosSteadyFlightODE` and import other stuff from phases.py

Reply:
The `DymosSteadyFlightODE` has very few changes from the `SteadyFlightPhase`, so I don't think it's worth keeping. I'm getting rid of it too